Gemfile

```diff
@@ -7,17 +7,17 @@ else
 end
 
 group :development, :unit_tests do
-  gem 'rake',                                             '< 11.0.0'
-  gem 'rspec-puppet', '~> 2.6.0',                         :require => false
+  gem 'rake'
+  gem 'rspec-puppet',                                     :require => false
   gem 'rspec-puppet-facts',                               :require => false
   gem 'rspec-mocks',                                      :require => false
-  gem 'puppetlabs_spec_helper', '>= 2.11.0',              :require => false
-  gem 'puppet-lint', "~> 2.0",                            :require => false
-  gem 'json', "~> 1.8.3",                                 :require => false
-  gem 'json_pure', "~> 1.8.3",                            :require => false
+  gem 'puppetlabs_spec_helper',                           :require => false
   gem 'metadata-json-lint',                               :require => false
   gem 'puppet-blacksmith',                                :require => false
-  gem 'rest-client', ">= 1.7.4",                          :require => false
+  gem 'json',                                             :require => false
+  gem 'json_pure',                                        :require => false
```

Comment:
We don't directly use `json` or `json_pure`, they should probably be pulled in by whatever else needs them.  Maybe remove them?  I think same could be said for `rest-client`.

Reply:
Let's do this in another PR after this one passes